### ISSUE TITLE
Fullscreen is actually only halfscreen in Chrome

### ISSUE THREAD

**Issue by boldtrn** [Leaflet/Leaflet.fullscreen#46]
Hi,

I just tested the fullscreen plugin. It works perfectly in FireFox, but using Google Chrome results with this:
![screen shot 2015-12-27 at 18 02 58](https://cloud.githubusercontent.com/assets/1553525/12011188/50330830-acc4-11e5-82ad-6e902708a02a.png)

**Comment by jfirebaugh** [comment]
[The demo](http://leaflet.github.io/Leaflet.fullscreen/) works fine for me on Chrome 47.0.2526.106. Can you please provide a self-contained example that demonstrates the problem?

**Comment by boldtrn** [comment]
@jfirebaugh Well the demo works for me as well. I just saw that you updated the CDN (I was using the CDN), this fixed the pseudo fullscreen for me (this is exactly what I needed).

However, the fullscreen still does not work. I tried to reproduce the error, in a self-contained example (like the demo, just with the cdn), worked like the demo. So this is probably an interference with some other lib I use. I am happy with the pseudo fullscreen :+1:.
